src/method/method_invoke_java.cpp

```diff
@@ -187,6 +187,15 @@ namespace cubmethod
     value_unpacker.value = &returnval;
     value_unpacker.unpack (unpacker);
 
+    if (db_value_type (&returnval) == DB_TYPE_RESULTSET)
+      {
```

Comment:
returnval is always DB_TYPE_NULL by precediing code "db_make_null (&returnval)".

Reply:
returnval is set by unpack at the above
```
value_unpacker.value = &returnval;
value_unpacker.unpack (unpacker);
```

Reply:
Ok, I missed the unpack.